sdk/servicebus/Azure.Messaging.ServiceBus/tests/Transactions/TransactionLiveTests.cs

```diff
@@ -971,7 +971,7 @@ public async Task CrossEntityTransactionSessionProcessorRollback()
 
             await processorA.StartProcessingAsync();
             await tcs.Task;
-            await processorA.StopProcessingAsync();
+            await processorA.CloseAsync();
```

Comment:
This test was only incidentally passing before due to the bug being fixed here - the session lock would get released when an accept session call timed out so on L977 we would be able to lock the session.